lms/envs/common.py

```diff
@@ -2686,28 +2686,31 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # .. setting_default: dict of settings
 # .. setting_description: Stores all the settings used by block structures and block structure
 #   related tasks. See BLOCK_STRUCTURES_SETTINGS[XXX] documentation for details of each setting.
-#   For more information, check https://openedx.atlassian.net/browse/TNL-5041.
+#   For more information, check https://github.com/edx/edx-platform/pull/13388.
```

Comment:
Hello @nizarmah.  I’m not going to have time for a while to get to this. Feel free to find anyone to open the links and double-check they are all what you meant to use, or we could just merge. In general this all looks fine.

Separately, we should probably start a discuss post to determine whether or not it is useful to have a separate annotation for this or not. If so, what should it be called. If not, should we just document things to consider including in the description? Should toggles and settings match here? Originally, I think the toggle annotation was planned for calling out a DEPR ticket, but I don’t think it is often being used for that.

Reply:
> Feel free to find anyone to open the links and double-check they are all what you meant to use, or we could just merge. In general this all looks fine.

I'll double-check over the weekend.
I'm confident of them, but a quick self review doesn't hurt 😃 

> Separately, we should probably start a discuss post to determine whether or not it is useful to have a separate annotation for this or not. If so, what should it be called.

Sure, I guess we can start a forum discussion regarding that. Possibly "toggle_prs"? Or something that would allow us to write down the issue number instead of the whole url, unless you think the url is more helpful to just copy-paste.

> If not, should we just document things to consider including in the description

This is going to be hard, especially since the number of toggles is still pretty high.
Maybe we can have the community create a Jira ticket, which can be linked to. And then, people can, there, link pull requests, link documentation, link other private tickets, and/or post comments?

> Originally, I think the toggle annotation was planned for calling out a DEPR ticket, but I don’t think it is often being used for that.

I guess, we might as well move the conversation to a forum post and speak our mind there? That way everyone can join in? Would you like to start the forum discussion regarding this issue, or would you prefer I do so?

Reply:
Yes. Let's move this to the forum. Please start it. Heading out early today. :) Thank you!